### PR TITLE
Create About Us Page

### PR DESCRIPTION
**Pull request by kevinquayle**
Created the about us page and added navigation to it from the side bar.

I was unable to figure out how to store and reference a local image within a card, but the image is stored in components > component-images. If someone could help with that, that would be great!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ const App = () => {
           {/* General Pages */}
           <Route path="/" element={<HomePage />} />
           <Route path="/modules" element={<ModuleSelectionPage />} />
+          <Route path="/about-us" element={<AboutUsPage />} />
```
Comment (masakiasanuma) [inline src/App.js:24]: Very minor detail, but can you change this route to "/about"?

File: src/components/AboutUsPage.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
+import styled from 'styled-components';
+import Card from 'react-bootstrap/Card';
+import { useNavigate } from 'react-router-dom';
+
```
Comment (masakiasanuma) [inline src/components/AboutUsPage.js:4]: To use a local image, you'd have to import it first like this:
`import samplePortrait from './component-images/sample-portrait.png';`

File: src/components/AboutUsPage.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            </ParagraphWrap>
+
+            <SubHeading>
+                Meet Our Team
+            </SubHeading>
+
+            <PortraitWrap>
+
+                <Card style={{ width: '20%', margin: '2.5%', display: 'inline-block' }}>
+                    <Card.Img variant="top" src="https://cdn-icons-png.flaticon.com/512/880/880974.png" />
```
Comment (masakiasanuma) [inline src/components/AboutUsPage.js:31]: Then reference it here, like this:
`src={samplePortrait}`

**Comment by masakiasanuma** [comment]
This is my personal preference, but can you make two separate sections for the client (Dr. Emily Blum, potentially the others) and the junior design team (us)? If it's too much work then we can leave it as is.

**Review by masakiasanuma — CHANGES_REQUESTED** [review]
Added minor comments, everything else looks good!

**Comment by kevinquayle** [comment]
I made the requested changes.

**Review by masakiasanuma — APPROVED** [review]
Thanks for the changes, approved.

**Review by valistelea — APPROVED** [review]
Looks great!

**Review by KhelanM — APPROVED** [review]

**Pull request merged**